tests/benchmarks/power_consumption/qdec/testcase.yaml

```diff
@@ -0,0 +1,24 @@
+common:
+  tags: ci_tests_benchmarks_current_consumption ppk_power_measure sensors
+  harness: pytest
+
+tests:
+  benchmarks.power_consumption.qdec_nrf54l:
+    platform_allow:
+      - nrf54l15pdk/nrf54l15/cpuapp
+    integration_platforms:
+      - nrf54l15pdk/nrf54l15/cpuapp
+    harness_config:
+      fixture: gpio_loopback
+      pytest_root:
+        - "${CUSTOM_ROOT_TEST_DIR}/test_measure_power_consumption.py::test_thread_suspend_resume_gpio_54L"
```

Comment:
gpio?

Reply:
The limits set in this function are universal, there is no point in creating a new function that differs only in name